docs/lcb/notes/feature-canvas_effect_for_rect.md

```diff
@@ -0,0 +1,41 @@
+# LiveCode Builder Host Library
+## Canvas library
+
+New syntax has been added to the Canvas library to allow specifying the drawing boundary
```

Comment:
@livecodeian I'm thinking this patch should also amend `docs/lcb/feature-effect_only_layer.md` to note the new layer syntax also applies there